src/scores/continuous.py

```diff
@@ -52,39 +64,41 @@ def mse(fcst, obs, reduce_dims=None, preserve_dims=None, weights=None):
     return _mse
 
 
-def mae(fcst, obs, reduce_dims=None, preserve_dims=None, weights=None):
-    """**Needs a 1 liner function description**
+def mae(fcst: OBS_TYPE, obs: OBS_TYPE, reduce_dims: Tuple[str] = None, preserve_dims: Tuple[str] = None, weights=None):
```

Comment:
No return type hint is specified. reuce_dims and preserve_dims could be strings, tuples (of strings), sets (of strings) or lists (of strings).

Reply:
Removed as per this comment: https://github.com/nci/scores/pull/16#discussion_r1294245090